wiki/plugins/images/settings.py

```diff
@@ -12,5 +12,3 @@
 ANONYMOUS = getattr(django_settings, 'WIKI_ATTACHMENTS_ANONYMOUS', False)
 
 SLUG = 'images'
-
-APP_LABEL = 'wiki'
```

Comment:
ohhhhh, I see - this setting is gone, so I'm assuming the explicit `app_label` are gone b/c the default is now correct?